go.mod

```diff
@@ -4,12 +4,10 @@ go 1.15
 
 require (
 	github.com/go-logr/logr v0.1.0
-    	k8s.io/api v0.17.14
```

Comment:
This comes from running `go mod tidy`, right? Can we mention that in the commit message?

Reply:
yes, when ran the go mod it removed some not used things, will mention